### PR TITLE
Add new image component

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Recreating the image component to:
* Use the JS models available
* Work out the sizes automatically
* Decouple it from the grid as it can't be used elsewhere if it is.

## What does it look like?
WIP

## For interface components
- [x] Browser testing - Have you checked the component in our supported browsers
- [x] No javascript - Have you checked the component with javascript disabled
- [x] A11y testing - Have you run pa11y against the component and fixed/verified all errors, warnings and notices?

## Have the following been considered/are they needed?

- [x] Tests?
- [ ] Docs?

### CODE REVIEW

File: server/views/components/image/index.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+export const name = 'Image';
+export const handle = 'Image';
+const image = new Picture({
+  contentUrl: 'https://wellcomecollection.files.wordpress.com/2016/10/featured-image-dans-blog2.jpg',
+  caption: 'An edited frame from Animating the Body',
+  width: 1380,
+  height: 1104
+}).toJS();
+const sizes = [300, 600, 960, 1338];
+export const context = { image, sizes };
```
Comment (jamesgorrie) [inline server/views/components/image/index.config.js]: @gestchild This was what I was talking about when saying we can use the JS models to create the objects for the templates...

**Comment by jamesgorrie** [comment]
This CR is not going to be about integrating this component into the website as I can do that as a next step - I just want to get a consensus that the figure component is doing what it should.

File: server/views/components/image/index.njk [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<figure class="figure
+               {{ ' figure--full' if full }}
```
Comment (davidpmccormick) [inline server/views/components/image/index.njk]: I think this will output on multiple lines. Is that OK for our supported browsers (I've no idea).

Comment (jamesgorrie) [inline server/views/components/image/index.njk]: It does annoyingly, and is fine.
Using the `{{- blahblah -}}` syntax seems to screw up fractal.

I'll file a bug report.

**Pull request merged**